sdg/Indicator.py

```diff
@@ -234,3 +234,65 @@ def translate_data(text):
 
         # Finally place the translation for later access.
         self.translations[language] = indicator
+
+    def is_complete(self):
+        """Decide whether this indicator can be considered "complete".
+
+        Returns
+        -------
+        boolean
+            True if the indicator can be considered "complete", False otherwise.
+        """
```

Comment:
Any specific reason we need these comment blocks? The purpose of the function and its return type seems to be self-explanatory from the name

Reply:
@SavvasStephanides I try to always do doc blocks regardless (though sometimes I forget). It's not a huge deal, but if we ever do automatically-generated documentation, we'll be glad to have them. And in general it's better to have them than not.

Reply:
Fair enough. Just be consistent and remember to update when appropriate.

Reply:
@SavvasStephanides I should make clear that I'm far from a Python expert, and would be happy to adopt whatever coding and documentation standards we agree on. And I definitely agree that every code change should always include any necessary documentation changes.